### ISSUE TITLE
Getting only 30days data

### ISSUE THREAD

**Issue by rishav-rohit** [Chitz/BSE-Stock-Quotes#2]
not able to get data for more than 30days.

**Comment by Chitz** [comment]
Yeah, I haven't extended it to more than 30 days data at a time. Might do it soon. 

A work around for your problem could be to run the script again for the next month i.e start date would be + 1 the end date (or related logic). Hope this works for you. :)
